ec_api/apps/api_proxy/tests/test_views.py

```diff
@@ -49,3 +62,15 @@ def test_email_not_in_candiate_response(authorized_client, postcode_response):
         assert candidate["person"]["absolute_url"]
     with pytest.raises(KeyError):
         assert candidate["person"]["photo_url"]
+
+
+def test_address_picker(authorized_client, address_picker_response):
```

Comment:
When you want to use a fixture in a test, but don't actually need to use the return value in your test, you can use the `usefixtures` decorator:
```
@pytest.mark.usefixtures("address_picker_response")
def test_address_picker(authorized_client):
    resp_json = authorized_client.get("/api/v1/postcode/GL51NA/").json()
    assert len(resp_json["addresses"]) == 10
    assert resp_json["dates"] == []
```
Doesn't make any real difference but feel like it is a bit more explicit this way?